client/src/app/core/users/user.model.ts

```diff
@@ -133,4 +133,19 @@ export class User implements UserServerModel {
   isUploadDisabled () {
     return this.videoQuota === 0 || this.videoQuotaDaily === 0
   }
+
+  isAutoBlocked () {
+    return this.role === UserRole.USER && this.adminFlags !== UserAdminFlag.BYPASS_VIDEO_AUTO_BLACKLIST
+  }
+
+  hasNoQuotaLeft () {
```

Comment:
What about daily quota?

Reply:
In a moment I thought it was useless for the daily quota because it would be reset in the next 24 hours. 
You're right, I'm gonna add an helper and alert for the the daily quota too.